orbit/cmd/orbit/orbit.go

```diff
@@ -139,6 +139,12 @@ func main() {
 			Usage:   "Launch Fleet Desktop application (flag currently only used on darwin)",
 			EnvVars: []string{"ORBIT_FLEET_DESKTOP"},
 		},
+		&cli.BoolFlag{
+			Name:    "disable-kickstart-softwareupdated",
+			Usage:   "Disable periodic execution of 'launchctl kickstart -k softwareupdated' on macOS",
+			EnvVars: []string{"ORBIT_FLEET_DISABLE_KICKSTART_SOFTWAREUPDATED"},
+			Hidden:  true,
+		},
```

Comment:
I figured this could come in handy if running the command periodically ever causes any issues (or if macOS fixes the `softwareupdated` hanging issue at some point). Let me know if you don't like the idea, I can remove it (or if you think it shouldn't be hidden - I didn't think this was worth adding as user-visible option, more like a support one just in case).